e2etest/zt_basic_copy_sync_remove_test.go

```diff
@@ -0,0 +1,362 @@
+// Copyright © Microsoft <wastore@microsoft.com>
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package e2etest
+
+import (
+	"testing"
+)
+
+// ================================  Copy And Sync: Upload, Download, and S2S  =========================================
+func TestBasic_CopyUploadSingleBlob(t *testing.T) {
+	RunScenarios(
+		t,
+		eOperation.CopyAndSync(),
+		eTestFromTo.AllUploads(),
+		eValidate.AutoPlusContent(),
+		params{
+			recursive: true,
+		},
+		nil,
+		testFiles{
+			defaultSize: "1K",
+			shouldTransfer: []interface{}{
+				folder(""),
+				f("file1.txt"),
+			},
+		})
+}
+
+func TestBasic_CopyUploadEmptyBlob(t *testing.T) {
+	RunScenarios(
+		t,
+		eOperation.CopyAndSync(),
+		eTestFromTo.AllUploads(),
+		eValidate.Auto(),
+		params{
+			recursive: true,
+		},
+		nil,
+		testFiles{
+			defaultSize: "0K",
+			shouldTransfer: []interface{}{
+				folder(""),
+				f("file1.txt"),
+			},
+		})
+}
+
+//func TestBasic_CopyUploadLargeBlob(t *testing.T) {
```

Comment:
Do we have an existing environment variable to flip for long running jobs? Perhaps instead of commenting this test out, you can skip it, and have it be less easily forgotten for a developer.

Reply:
As of now we don't have a flip switch for skipping the test case. In fact we have to write a sub-framework within E2E test framework to accommodate for large file tests. Let me remove them